AUTHORS.rst

```diff
@@ -11,3 +11,4 @@ Contributors
 ~~~~~~~~~~~~
 
 * Khaled Porlin - https://github.com/porlin72
+* [Agam Dua](https://github.com/agamdua)
```

Comment:
I wanted to ask if I can do this for all? It just renders nicer. 

I don't mind switching it back or changing all. you just got to it before I could ask in a comment :wink: